src/app/person/person.hpp

```diff
@@ -3,18 +3,26 @@
 #include <memory>
 #include <string>
 
+enum class Gender
+{
+    Male,
+    Female
+};
```

Comment:
Please provide also function "toString" which will translate Gender enum to string. 
You can find toString example in alergen.cpp

Reply:
Done